### ISSUE TITLE
Adjust regular CORDEX grids

### ISSUE THREAD

**Issue by jesusff** [WCRP-CORDEX/domain-tables#5]
We need to adjust these tables to the new naming -50i, -25i, -12i.

There are also mismatches at the borders:
https://github.com/WCRP-CORDEX/domain-tables/blob/d8d54ec0d2654bc83031b8f32f3688bc4faf0002/regular.csv?plain=1#L14-L15

To be able to do an upscaling without interpolation, each cell in -50i should be full of cells of the -25i and -12i counterparts.

![CORDEX_regular_grids](https://github.com/WCRP-CORDEX/domain-tables/assets/8810015/736bb25f-bdfe-44be-b2ef-e127f6d24ad1)

If there is no specific library or tool interpreting the "ll_lon" naming, it seems more intuitive to keep the "west" header in the [old specs](https://is-enes-data.github.io/cordex_archive_specifications.pdf).

**Comment by larsbuntemeyer** [comment]
Good catch! These numbers are actually taken from the [archive specs](https://is-enes-data.github.io/cordex_archive_specifications.pdf) for my convenience to have a machine readable table. I don't know if these numbers have ever been used, the CORDEX-CORE data was published on the AFR-22 grid as far as i know...

Actually, i don't use this file anymore, since i have merged all domains in one [table](https://github.com/WCRP-CORDEX/domain-tables/blob/main/rotated-latitude-longitude.csv) for use in my [codes](https://py-cordex.readthedocs.io/en/latest/notebooks/introduction.html). However, i have to think about restructuring this anyway since some domains seem to change in CMIP6 downscaling.

**Comment by jesusff** [comment]
OK, in the merged file I understand the "ll_lon" notation. Let's keep it.
I think it is useful to have them stored here, so they can be updated and tracked. The CSV renderer in github can also be used to filter easily e.g. the "-22i" ones, or those for "AFR". We could even generate automatically from them some geojson files, which are also [rendered on a map](https://github.com/jesusff/domains/blob/main/data/EUR-11.geojson) or be used to display the domain borders on a global map.

**Comment by gnikulin** [comment]
Yes, we need to synchronise the regular grids at the 3 resolution, not so complex task.
